packages/wallets/src/components/WalletSuccess/WalletSuccess.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { ReactNode } from 'react';
```

Comment:
is this the same component as WalletActionScreen, but with one more button? Worth just adding props for extra button to other component instead?

Reply:
There other differences, like title size and description are different, gap between the title and description is also different. Let see what i can do